### ISSUE TITLE
Fix .well-known response and 3rd party domains

### ISSUE THREAD

**Issue by SebastianZimmeck** [privacy-tech-lab/gpc-optmeowt#464]
I am currently using v5.0.1 and I am am realizing that both on Chrome and Firefox the .well-known response is not shown.

Here is the result for nytimes.com

<img width="326" alt="Screenshot 2024-05-25 at 7 23 10 PM" src="https://github.com/privacy-tech-lab/gpc-optmeowt/assets/11034003/e16bef34-c6fa-4a64-9212-4cdcdcb886dc">

The same issue is present on the following sites:

- https://globalprivacycontrol.org/
- https://global-privacy-control.glitch.me/
- https://www.theatlantic.com/

So, it seems like displaying the .well-known is globally failing.

Can you look into that @Mattm27?

**Comment by Mattm27** [comment]
I have looked into this issue on my end to see if it was persisting and the well-known response is shown on my end for both Firefox and Chrome. 

Chrome:
![Uploading Screenshot 2024-05-25 at 7.48.12 PM.png…]()

Firefox:

**Comment by Mattm27** [comment]
I have looked into this issue on my end to see if it was persisting and the well-known response is shown on my computer for both Firefox and Chrome.

**Comment by SebastianZimmeck** [comment]
Thanks, @Mattm27!

Did you update to v5.0.1 from the Chrome Web Store (i.e., are you using the dist version)? What OS etc. are you using?

For example, I am using v5.0.1 from the Chrome Web Store with the out of the box current Chrome (Version 125.0.6422.113 (Official Build) (arm64)) on most recent macOS, and under Website Response, "gpc:true}" is not shown, but it exists.

Can you post screenshots?

<img width="1426" alt="Screenshot 2024-05-25 at 8 01 37 PM" src="https://github.com/privacy-tech-lab/gpc-optmeowt/assets/11034003/acb71ce2-e127-4ec7-b3ab-9ca7679a950d">

<img width="1432" alt="Screenshot 2024-05-25 at 8 04 51 PM" src="https://github.com/privacy-tech-lab/gpc-optmeowt/assets/11034003/f6373f33-f933-49c9-8843-0de4108f1bc5">

**Comment by Mattm27** [comment]
Yes I updated to v5.0.1 from the Chrome web store and am running mac OS v14.3.1

Here are screenshots:

Chrome:
<img width="489" alt="Screenshot 2024-05-25 at 9 53 55 PM" src="https://github.com/privacy-tech-lab/gpc-optmeowt/assets/144485976/905fd422-9b5d-4ca8-b051-0558d34c5720">

<img width="351" alt="Screenshot 2024-05-25 at 7 50 16 PM" src="https://github.com/privacy-tech-lab/gpc-optmeowt/assets/144485976/89667692-08fd-4a37-852a-5887d9e78bf7">


Firefox:
<img width="343" alt="Screenshot 2024-05-25 at 7 50 31 PM" src="https://github.com/privacy-tech-lab/gpc-optmeowt/assets/144485976/ff5d51a1-f396-4b8a-ad50-48739674ede8">

I am not sure why there would be discrepancies with this behavior between devices.

**Comment by SebastianZimmeck** [comment]
This is resolved. It seems the reason this occurs is because when I updated OptMeowt, I did not enable the required permissions. 

<img width="594" alt="Screenshot 2024-05-26 at 12 09 54 PM" src="https://github.com/privacy-tech-lab/gpc-optmeowt/assets/11034003/812d94e7-655f-494c-91f1-733385be28f9">

In more detail:

Because of just doing the update instead of a reinstall, I never saw the permission popup. I only saw it once I navigated to the OptMeowt settings. Then, I enabled it. That did not solve the issue immediately, though. When I retried loading the site, it still did not work. Then, I cleared my local browsing cache, uninstalled OptMeowt, and reinstalled it. After a bit of reloading and cache clearing, that did it. Now, it is working for me in both Chromium (Chrome and Opera) and Firefox. As an aside, this issue did not only affect the Website Response but also the 3rd Party Domains, which were not displayed either. It is still the case for some sites that I need to reload them to get the Website Response but also the 3rd Party Domains, but that is OK for the time being.
